README.md

```diff
@@ -1,12 +1,16 @@
-# gotrue-js
+# gotrue-js library
 
 [![Build Status](https://travis-ci.org/netlify/gotrue-js.svg?branch=master)](https://travis-ci.org/netlify/gotrue-js)
 
-This is a JS client library for the [GoTrue](https://github.com/netlify/gotrue) API.
+This is a JavaScript client library for the [GoTrue](https://github.com/netlify/gotrue) API.
 
-It lets you signup and authenticate users and is a building block for constructing
+It lets you create and authenticate users and is a building block for constructing
 the UI for signups, password recovery, login and logout.
 
+Notice that any functions that interact with browser `window object`, such as `localStorage`, need to be tested in the browser instead of in the REPL.
```

Comment:
Rather than state this in the beginning, we should note which calls are browser only.

Reply:
I would just remove this part for now.